### DISCUSSION TITLE
How to use default_classes property together with removing page default padding?

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#2562] [Q&A] [answered]
### Question

Hi All,

I realized that when I use the `default_classes` property for a label, I cannot remove the default page paddings. They are added back for some reason.

Below is the code that I tried:
```python
ui.query('.nicegui-content').classes('p-0')
ui.label().default_classes('text-white')
with ui.row().classes('h-screen w-screen bg-blue'):
    ui.label("This is for testing")
```

If I don't use the second line, the code works fine. Any idea why this is happening and what would be the correct way to use this?

Thanks a lot,
Anindya

**Comment by falkoschindler** [comment, marked as answer]
Hi @Anindya088!

With the line
```py
ui.label().default_classes('text-white')
```
you're creating a label instance, which causes the white gap at the top.

Instead you can call `default_classes()` on the _class_ instead of an instance:
```py
ui.label.default_classes('text-white')
```

**Reply by Anindya088** [reply]
Hi @falkoschindler , thanks so much. Sorry that was silly, I should have understood that.
Thanks so much for the quick help.

Regards,
Anindya
